apps/experiments/migrations/0032_load_synthetic_voices.py

```diff
@@ -1,30 +1,15 @@
 # Generated by Django 4.2 on 2023-09-28 07:13
 import os
-import json
 from django.db import migrations
+from apps.utils import migration_utils
 
 
-def load_synthetic_voices(apps, schema_editor):
+def drop_synthetic_voices(apps, schema_editor):
```

Comment:
This should be `load_synthetic_voices`

Reply:
did you change this - looks like it is still 'drop' instead of 'load'

Reply:
I did, and double checked it :raised_eyebrow:, but....it's gone.. :open_mouth: